tools/update-eslint.sh

```diff
@@ -17,7 +17,11 @@ npm install --global-style --no-bin-links --production --no-package-lock eslint@
 cd node_modules/eslint
 
 npm install --no-bin-links --production --no-package-lock eslint-plugin-markdown@next
-cd ../..
+npm install --no-bin-links --production --no-package-lock babel-eslint
+
+cd node_modules
+ln -s .. eslint
```

Comment:
Do any of you think that this `ln -s` can create problems in Windows machines?

Reply:
On windows that would have to be `mklink` instead: https://superuser.com/questions/182553/does-windows-have-the-ln-s-or-equivalent

Reply:
This is a bash script that we run to update eslint. I create that symbolic link because otherwise babel-eslint would not be able to resolve eslint itself. That link is currently committed to git.

Reply:
I don't think symlinks work by default on Windows because they require special permissions. I tested it by creating a symlink to a file on a Linux computer and then cloning the repo on Windows. Instead of the link, there is a text file with the destination of the link inside.

Reply:
To make babel-eslint work correctly without symbolic links, I think we need to install both eslint and babel-eslint in `tools/node_modules`.

Reply:
@targos I agree. Should I do that? I tried to preserve the original structure as much as possible.

Reply:
@mcollina https://github.com/nodejs/node/pull/17820